samples/compose-samples/src/main/java/com/squareup/sample/compose/textinput/TextInputWorkflow.kt

```diff
@@ -5,30 +5,38 @@ import com.squareup.sample.compose.textinput.TextInputWorkflow.State
 import com.squareup.workflow1.Snapshot
 import com.squareup.workflow1.StatefulWorkflow
 import com.squareup.workflow1.action
+import com.squareup.workflow1.ui.TextController
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 
+@OptIn(WorkflowUiExperimentalApi::class)
 object TextInputWorkflow : StatefulWorkflow<Unit, State, Nothing, Rendering>() {
 
   data class State(
```

Comment:
One wrinkle: I originally tried to use a `State` like this:

```kotlin
  data class State(
    val textA: TextController(""),
    val textB: TextController(""),
    val showingTextA: Boolean = true
  )
```

And then just swap which one was included in the rendering. That doesn’t work because `remember {}` only executes once, so the composition always holds the first `TextController` it saw.